utils/terrascript_client.py

```diff
@@ -1248,6 +1249,21 @@ def populate_tf_resource_sqs(self, resource, namespace_info):
                 if self._multiregion_account_(account):
                     values['provider'] = 'aws.' + region
                 values.update(defaults)
+                kms_master_key_id = values.pop('kms_master_key_id', None)
+                if kms_master_key_id:
```

Comment:
Shouldn't we check only agains `None`? Anything else should be considered that the value is there...